siliconcompiler/schema/schema_cfg.py

```diff
@@ -3506,6 +3484,177 @@ def schema_constraint(cfg):
 
     return cfg
 
+############################################
+# Server Configuration
+############################################
+
+def schema_server(cfg):
+
+    scparam(cfg, ['server', 'jobscheduler'],
```

Comment:
Like Peter mentioned, may be worth keeping this under `option`. It may also be worth considering other names for the top-level key, as the job scheduling parameters will work the same in both local and server runs. (Depending on the server)

Maybe `cluster`, `grid`, or `scheduler`?

Reply:
You are right, I forgot that we support running jobs on premises with slurm and without the -remote flag. Since slurm/lsf is such an important feature, I think it warrants continuing this path. I will change the name to 'scheduler'.